fidesops.toml

```diff
@@ -1,34 +1,37 @@
 PORT=8080
 
 [database]
-SERVER="db"
-USER="postgres"
-PASSWORD="216f4b49bea5da4f84f05288258471852c3e325cd336821097e1e65ff92b528a"
-DB="app"
-TEST_DB="test"
-ENABLED=true
+SERVER = "db"
+USER = "postgres"
+PASSWORD = "216f4b49bea5da4f84f05288258471852c3e325cd336821097e1e65ff92b528a"
+DB = "app"
+TEST_DB = "test"
+ENABLED = true
 
 [redis]
-HOST="redis"
-PASSWORD="testpassword"
-PORT=6379
-CHARSET="utf8"
-DEFAULT_TTL_SECONDS=604800
-DB_INDEX=0
-ENABLED=true
+HOST = "redis"
+PASSWORD = "testpassword"
+PORT = 6379
+CHARSET = "utf8"
+DEFAULT_TTL_SECONDS = 604800
+DB_INDEX = 0
+ENABLED = true
 
 [security]
-APP_ENCRYPTION_KEY="OLMkv91j8DHiDAULnK5Lxx3kSCov30b3"
-CORS_ORIGINS=["http://localhost", "http://localhost:8080", "http://localhost:3000", "http://localhost:3001"]
-ENCODING="UTF-8"
-OAUTH_ROOT_CLIENT_ID="fidesopsadmin"
-OAUTH_ROOT_CLIENT_SECRET="fidesopsadminsecret"
-DRP_JWT_SECRET="secret"
+APP_ENCRYPTION_KEY = "OLMkv91j8DHiDAULnK5Lxx3kSCov30b3"
+CORS_ORIGINS = [ "http://localhost", "http://localhost:8080", "http://localhost:3000", "http://localhost:3001",]
```

Comment:
This change feels arbitrary, and would just mess with the `git blame`. Please revert 😄

Reply:
These spaces will be added anyway, when the fidesops.toml gets overridden when `client_id` is generated. Probably something with `toml.dump`- https://github.com/ethyca/fidesops/pull/541/files#diff-9aca0346a2a86990290c4e236109ceab5ff39557687d38c0aefd74da1659f3edR416. Should I still update?